src/environment.js

```diff
@@ -0,0 +1,43 @@
+// Fallback to rinkeby
+const FALLBACK_CHAIN_ID = 4
+
+const ENV_VARS = {
+  BUILD() {
```

Comment:
I'm not sure if there was a good reason these had to be functions in the court-dashboard, but it might be simpler if they were just constants in an object.

Reply:
IIRC the idea was to do any type conversion / filtering in these functions (e.g. below for `DEFAULT_CHAIN_ID`).

Reply:
It seemed liked a flexible pattern from the Court. Unless you feel strongly I'll leave it as is for now.